packages/state-transition/src/block/processExecutionLayerExit.ts

```diff
@@ -0,0 +1,50 @@
+import {byteArrayEquals} from "@chainsafe/ssz";
+import {electra} from "@lodestar/types";
+import {ETH1_ADDRESS_WITHDRAWAL_PREFIX, FAR_FUTURE_EPOCH} from "@lodestar/params";
+
+import {isActiveValidator} from "../util/index.js";
+import {CachedBeaconStateElectra} from "../types.js";
+import {initiateValidatorExit} from "./index.js";
+
+export function processExecutionLayerExit(state: CachedBeaconStateElectra, exit: electra.ExecutionLayerExit): void {
+  if (!isValidExecutionLayerExit(state, exit)) {
+    return;
+  }
+
+  const {epochCtx} = state;
+  const validatorIndex = epochCtx.getValidatorIndex(exit.validatorPubkey);
+  const validator = validatorIndex !== undefined ? state.validators.get(validatorIndex) : undefined;
+  if (validator === undefined) {
+    throw Error("Internal error validator=undefined for a valid execution layer exit");
+  }
+  initiateValidatorExit(state, validator);
+}
+
+export function isValidExecutionLayerExit(state: CachedBeaconStateElectra, exit: electra.ExecutionLayerExit): boolean {
```

Comment:
should we return validators here so that consumer doesn't need to get it again? then this function return `Validator | null`